app/clients/letter/dvla.py

```diff
@@ -0,0 +1,71 @@
+import time
+
+import boto3
+import jwt
+import requests
+from flask import current_app
+
+
+class SSMProperty:
+    def __init__(self, key):
+        self.key = key
+
+    def __get__(self, obj, objtype=None):
+        return obj.ssm_client.get_parameter(Name=self.key, WithDecryption=True)["Parameter"]["Value"]
+
+    def __set__(self, obj, value):
+        # this errors if the parameter doesn't exist yet in SSM as we haven't supplied a `Type`
+        # this is fine for our purposes, as we'll always want to pre-seed this data.
+        obj.ssm_client.put_parameter(Name=self.key, Value=value, Overwrite=True)
```

Comment:
i definitely think caching should come in, 100%, since this pr is hanging around a little longer than i intended i'll add it in

Reply:
I don't think there's a good naming solution here. Just use proper function/methods that clearly say "work is happening". Especially since SSM is (currently) not a thing that we use much, I think hiding the fact that we're read/writing to an actual secrets store is not the right thing to do.